src/styles/blocks/container.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  max-width: 1199px;
+
+  &__box {
+    display: flex;
```

Comment:
I think display flex is rebundant here.

Reply:
i thought it would be better to center with flex since the other way i know is line-height :)

Reply:
Yes but in checklist they have mentioned that use grid and try not to use flex.
I think you have changed it to text-align:center and removed flex for centering the text inside the block.

Reply:
look into `justify-items` and `align-items`
https://css-tricks.com/snippets/css/complete-guide-grid/